Index: LAB3/Lab3/src/Aims.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LAB3/Lab3/src/Aims.java b/LAB3/Lab3/src/Aims.java
deleted file mode 100644
--- a/LAB3/Lab3/src/Aims.java	(revision 0a1ea7aaa5add89feab02532787446123f509790)
+++ /dev/null	(revision 0a1ea7aaa5add89feab02532787446123f509790)
@@ -1,23 +0,0 @@
-package Lab3.src;
-
-public class Aims {
-    public static void main(String[] args) {
-        Cart anOrder = new Cart();
-
-        DigitalVideoDisc dvd1 = new DigitalVideoDisc("The Lion King", "Animation", "Roger Allers", 87, (float)19.95);
-        DigitalVideoDisc dvd2 = new DigitalVideoDisc("Star Wars", "Science Fiction", "George Lucas", 124, (float)24.95);
-        DigitalVideoDisc dvd3 = new DigitalVideoDisc("Aladdin", "Animation", "John Musker", 90, (float)18.99);
-
-        anOrder.addDigitalVideoDisc(dvd1);
-        anOrder.addDigitalVideoDisc(dvd2);
-        anOrder.addDigitalVideoDisc(dvd3);
-
-        anOrder.addDigitalVideoDisc(dvd1,dvd2,dvd3);
-
-        System.out.println("Total cost is: " + anOrder.totalCost());
-
-        anOrder.removeDigitalVideoDisc(dvd1);
-
-        System.out.println("Total cost is: " + anOrder.totalCost());
-    }
-}
\ No newline at end of file
Index: LAB3/Lab3/src/DigitalVideoDisc.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LAB3/Lab3/src/DigitalVideoDisc.java b/LAB3/Lab3/src/DigitalVideoDisc.java
deleted file mode 100644
--- a/LAB3/Lab3/src/DigitalVideoDisc.java	(revision 0a1ea7aaa5add89feab02532787446123f509790)
+++ /dev/null	(revision 0a1ea7aaa5add89feab02532787446123f509790)
@@ -1,35 +0,0 @@
-package Lab3.src;
-public class DigitalVideoDisc{
-    private String title;
-    private String category;
-    private String director;
-    private int length;
-    private float cost; 
-
-    public DigitalVideoDisc(String title, String category, String director, int length, float cost){
-        this.title = title;
-        this.category = category;
-        this.director = director;
-        this.length = length;
-        this.cost = cost;
-    }
-    
-
-    public String getTitle(){
-        return title;
-    }
-    public String getCategory(){
-        return category;
-    }
-    public String getDirector(){
-        return director;
-    }
-    public int length(){
-        return length;
-    }
-    public float getCost(){
-        return cost;
-    }
-
-
-}
\ No newline at end of file
Index: LAB3/Lab3/src/Cart.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LAB3/Lab3/src/Cart.java b/LAB3/Lab3/src/Cart.java
deleted file mode 100644
--- a/LAB3/Lab3/src/Cart.java	(revision 0a1ea7aaa5add89feab02532787446123f509790)
+++ /dev/null	(revision 0a1ea7aaa5add89feab02532787446123f509790)
@@ -1,48 +0,0 @@
-package Lab3.src;
-import java.util.LinkedList;
-
-public class Cart {
-    public static final int MAX_NUMBERS_ORDERED = 20;
-    LinkedList<DigitalVideoDisc> itemsOrdered = new LinkedList<>();
-
-    public void addDigitalVideoDisc(DigitalVideoDisc a) {
-        if (itemsOrdered.size() < MAX_NUMBERS_ORDERED) {
-            itemsOrdered.add(a);
-            System.out.println("The disc has been added.");
-            System.out.println("Cost: " + totalCost());
-        } else {
-            System.out.println("The cart is full!");
-        }
-    }
-
-    public double totalCost() {
-        double sum = 0;
-        for (int i = 0; i < itemsOrdered.size(); i++) {
-            sum += itemsOrdered.get(i).getCost();
-        }
-        return sum;
-    }
-
-    public void removeDigitalVideoDisc(DigitalVideoDisc a) {
-        if (itemsOrdered.remove(a)) {
-            System.out.println("The disc has been removed.");
-        } else {
-            System.out.println("The disc is not in the cart.");
-        }
-    }
-    /**
-    public void addDigitalVideoDisc(DigitalVideoDisc[] dvdList) {
-        for (int i = 0; i < dvdList.length; i++) {
-            addDigitalVideoDisc(dvdList[i]);
-        }
-    }
-    **/
-
-    //This work better
-    public void addDigitalVideoDisc(DigitalVideoDisc... dvds) {
-        for (DigitalVideoDisc dvd : dvds) {
-            this.addDigitalVideoDisc(dvd);
-        }
-    }
-
-}
\ No newline at end of file
